# A msg (signal) to tell the robot controller to move to next waypoint
Header header
bool if_ready